# Copyright (C) 2021 Aleksandr Migunov

# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 3 of the License, or
# (at your option) any later version.

# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
  
# You should have received a copy of the GNU General Public License
# along with this program; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

from PySide import QtCore, QtGui

class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(800, 600)
        self.centralwidget = QtGui.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout = QtGui.QGridLayout(self.centralwidget)
        self.gridLayout.setObjectName("gridLayout")
        self.label = QtGui.QLabel(self.centralwidget)
        self.label.setObjectName("label")
        self.gridLayout.addWidget(self.label, 0, 0, 1, 1)
        self.label_2 = QtGui.QLabel(self.centralwidget)
        self.label_2.setObjectName("label_2")
        self.gridLayout.addWidget(self.label_2, 0, 1, 1, 1)
        self.textEdit = QtGui.QTextEdit(self.centralwidget)
        self.textEdit.setObjectName("textEdit")
        self.gridLayout.addWidget(self.textEdit, 1, 0, 1, 1)
        self.textEdit_2 = QtGui.QTextEdit(self.centralwidget)
        self.textEdit_2.setObjectName("textEdit_2")
        self.gridLayout.addWidget(self.textEdit_2, 1, 1, 1, 1)
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtGui.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 800, 25))
        self.menubar.setObjectName("menubar")
        self.menuFile = QtGui.QMenu(self.menubar)
        self.menuFile.setObjectName("menuFile")
        self.menuOpen = QtGui.QMenu(self.menuFile)
        self.menuOpen.setObjectName("menuOpen")
        self.menuSave = QtGui.QMenu(self.menuFile)
        self.menuSave.setObjectName("menuSave")
        self.menuNew = QtGui.QMenu(self.menuFile)
        self.menuNew.setObjectName("menuNew")
        self.menuConvert_2 = QtGui.QMenu(self.menubar)
        self.menuConvert_2.setObjectName("menuConvert_2")
        self.menuTo_Ingush = QtGui.QMenu(self.menuConvert_2)
        self.menuTo_Ingush.setObjectName("menuTo_Ingush")
        self.menuTo_Avar = QtGui.QMenu(self.menuConvert_2)
        self.menuTo_Avar.setObjectName("menuTo_Avar")
        self.menuTo_Chechen = QtGui.QMenu(self.menuConvert_2)
        self.menuTo_Chechen.setObjectName("menuTo_Chechen")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtGui.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.actionTo_Ingush = QtGui.QAction(MainWindow)
        self.actionTo_Ingush.setObjectName("actionTo_Ingush")
        self.actionTo_Avar = QtGui.QAction(MainWindow)
        self.actionTo_Avar.setObjectName("actionTo_Avar")
        self.actionTo_Chechen = QtGui.QAction(MainWindow)
        self.actionTo_Chechen.setObjectName("actionTo_Chechen")
        self.actionExit = QtGui.QAction(MainWindow)
        self.actionExit.setObjectName("actionExit")
        self.actionWith_original = QtGui.QAction(MainWindow)
        self.actionWith_original.setObjectName("actionWith_original")
        self.actionIngush_with_original = QtGui.QAction(MainWindow)
        self.actionIngush_with_original.setObjectName("actionIngush_with_original")
        self.actionIngush_without_original = QtGui.QAction(MainWindow)
        self.actionIngush_without_original.setObjectName("actionIngush_without_original")
        self.actionAvar_with_original = QtGui.QAction(MainWindow)
        self.actionAvar_with_original.setObjectName("actionAvar_with_original")
        self.actionAvar_without_original = QtGui.QAction(MainWindow)
        self.actionAvar_without_original.setObjectName("actionAvar_without_original")
        self.actionChechen_with_original = QtGui.QAction(MainWindow)
        self.actionChechen_with_original.setObjectName("actionChechen_with_original")
        self.actionChechen_without_original = QtGui.QAction(MainWindow)
        self.actionChechen_without_original.setObjectName("actionChechen_without_original")
        self.actionOpen_original = QtGui.QAction(MainWindow)
        self.actionOpen_original.setObjectName("actionOpen_original")
        self.actionOpen_converted = QtGui.QAction(MainWindow)
        self.actionOpen_converted.setObjectName("actionOpen_converted")
        self.actionSave_original = QtGui.QAction(MainWindow)
        self.actionSave_original.setObjectName("actionSave_original")
        self.actionSave_converted = QtGui.QAction(MainWindow)
        self.actionSave_converted.setObjectName("actionSave_converted")
        self.actionNew_original = QtGui.QAction(MainWindow)
        self.actionNew_original.setObjectName("actionNew_original")
        self.actionNew_converted = QtGui.QAction(MainWindow)
        self.actionNew_converted.setObjectName("actionNew_converted")
        self.menuOpen.addAction(self.actionOpen_original)
        self.menuOpen.addAction(self.actionOpen_converted)
        self.menuSave.addAction(self.actionSave_original)
        self.menuSave.addAction(self.actionSave_converted)
        self.menuNew.addAction(self.actionNew_original)
        self.menuNew.addAction(self.actionNew_converted)
        self.menuFile.addAction(self.menuNew.menuAction())
        self.menuFile.addAction(self.menuOpen.menuAction())
        self.menuFile.addAction(self.menuSave.menuAction())
        self.menuFile.addSeparator()
        self.menuFile.addAction(self.actionExit)
        self.menuTo_Ingush.addAction(self.actionIngush_with_original)
        self.menuTo_Ingush.addAction(self.actionIngush_without_original)
        self.menuTo_Avar.addAction(self.actionAvar_with_original)
        self.menuTo_Avar.addAction(self.actionAvar_without_original)
        self.menuTo_Chechen.addAction(self.actionChechen_with_original)
        self.menuTo_Chechen.addAction(self.actionChechen_without_original)
        self.menuConvert_2.addAction(self.menuTo_Ingush.menuAction())
        self.menuConvert_2.addAction(self.menuTo_Avar.menuAction())
        self.menuConvert_2.addAction(self.menuTo_Chechen.menuAction())
        self.menubar.addAction(self.menuFile.menuAction())
        self.menubar.addAction(self.menuConvert_2.menuAction())

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        MainWindow.setWindowTitle(QtGui.QApplication.translate("MainWindow", "Converter", None, QtGui.QApplication.UnicodeUTF8))
        self.label.setText(QtGui.QApplication.translate("MainWindow", "<b>Original list</b>", None, QtGui.QApplication.UnicodeUTF8))
        self.label_2.setText(QtGui.QApplication.translate("MainWindow", "<b>Converted list</b>", None, QtGui.QApplication.UnicodeUTF8))
        self.menuFile.setTitle(QtGui.QApplication.translate("MainWindow", "File", None, QtGui.QApplication.UnicodeUTF8))
        self.menuOpen.setTitle(QtGui.QApplication.translate("MainWindow", "Open", None, QtGui.QApplication.UnicodeUTF8))
        self.menuSave.setTitle(QtGui.QApplication.translate("MainWindow", "Save", None, QtGui.QApplication.UnicodeUTF8))
        self.menuNew.setTitle(QtGui.QApplication.translate("MainWindow", "New", None, QtGui.QApplication.UnicodeUTF8))
        self.menuConvert_2.setTitle(QtGui.QApplication.translate("MainWindow", "Convert", None, QtGui.QApplication.UnicodeUTF8))
        self.menuTo_Ingush.setTitle(QtGui.QApplication.translate("MainWindow", "To Ingush", None, QtGui.QApplication.UnicodeUTF8))
        self.menuTo_Avar.setTitle(QtGui.QApplication.translate("MainWindow", "To Avar", None, QtGui.QApplication.UnicodeUTF8))
        self.menuTo_Chechen.setTitle(QtGui.QApplication.translate("MainWindow", "To Chechen", None, QtGui.QApplication.UnicodeUTF8))
        self.actionTo_Ingush.setText(QtGui.QApplication.translate("MainWindow", "To Ingush", None, QtGui.QApplication.UnicodeUTF8))
        self.actionTo_Avar.setText(QtGui.QApplication.translate("MainWindow", "To Avar", None, QtGui.QApplication.UnicodeUTF8))
        self.actionTo_Chechen.setText(QtGui.QApplication.translate("MainWindow", "To Chechen", None, QtGui.QApplication.UnicodeUTF8))
        self.actionExit.setText(QtGui.QApplication.translate("MainWindow", "Exit", None, QtGui.QApplication.UnicodeUTF8))
        self.actionWith_original.setText(QtGui.QApplication.translate("MainWindow", "With original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionIngush_with_original.setText(QtGui.QApplication.translate("MainWindow", "Ingush with original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionIngush_without_original.setText(QtGui.QApplication.translate("MainWindow", "Ingush without original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionAvar_with_original.setText(QtGui.QApplication.translate("MainWindow", "Avar with original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionAvar_without_original.setText(QtGui.QApplication.translate("MainWindow", "Avar without original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionChechen_with_original.setText(QtGui.QApplication.translate("MainWindow", "Chechen with original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionChechen_without_original.setText(QtGui.QApplication.translate("MainWindow", "Chechen without original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionOpen_original.setText(QtGui.QApplication.translate("MainWindow", "Open original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionOpen_converted.setText(QtGui.QApplication.translate("MainWindow", "Open converted", None, QtGui.QApplication.UnicodeUTF8))
        self.actionSave_original.setText(QtGui.QApplication.translate("MainWindow", "Save original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionSave_converted.setText(QtGui.QApplication.translate("MainWindow", "Save converted", None, QtGui.QApplication.UnicodeUTF8))
        self.actionNew_original.setText(QtGui.QApplication.translate("MainWindow", "New original", None, QtGui.QApplication.UnicodeUTF8))
        self.actionNew_converted.setText(QtGui.QApplication.translate("MainWindow", "New converted", None, QtGui.QApplication.UnicodeUTF8))

